### DISCUSSION TITLE
大佬迅雷云盘能搞不

### DISCUSSION THREAD

**Discussion by starsxxms1** [AlistGo/alist#341] [Q&A]
试了一下迅雷盘速度还行，不知道是否支持

**Comment by xhofe** [comment]
迅雷云盘网页端可操作的实在太少 请提供api文档

**Reply by viqbgrg** [reply]
> 

https://greasyfork.org/zh-CN/scripts/431256-%E8%BF%85%E9%9B%B7%E4%BA%91%E7%9B%98
你好,这个好像可以直链的
